framework/meta/src/cmd/contract/wasm_cargo_toml_data.rs

```diff
@@ -0,0 +1,55 @@
+use crate::cargo_toml_contents::CargoTomlContents;
+
+use super::sc_config::ContractVariantProfile;
+
+#[derive(Default, Debug, Clone, PartialEq, Eq)]
+pub struct WasmCargoTomlData {
+    pub name: String,
+    pub edition: String,
+    pub profile: ContractVariantProfile,
+    pub framework_version: String,
+    pub framework_path: Option<String>,
+    pub contract_features: Vec<String>,
+}
+
+impl WasmCargoTomlData {
+    pub fn change_package_edition(&mut self, cargo_toml_contents: &CargoTomlContents) {
```

Comment:
We don't need these setters, this is a simple struct.
It doesn't need to know about CargoTomlContents